sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1142,12 +1180,20 @@ public void testWriteUnknown() throws Exception {
         .withDatasetService(datasetService);
     datasetService.createDataset("project-id", "dataset-id", "", "");
     Pipeline p = TestPipeline.create(bqOptions);
+
+    TableSchema schema = new TableSchema()
```

Comment:
Why this change? Likewise for a couple of other changes below - they don't seem related to this PR.